itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosExternalTables.java

```diff
@@ -1680,4 +1682,104 @@ public void testDataCopyEndLog(boolean runCopyTasksOnTarget) throws Throwable {
     ctx.updateLoggers();
     appender.removeFromLogger(logger.getName());
   }
+
+  @Test
+  public void testSingleCopyTasksAtSource() throws Throwable {
+    testDataCopyEndLog(false);
+  }
+
+  @Test
+  public void testSingleCopyTasksAtTarget() throws Throwable {
+    testDataCopyEndLog(true);
+  }
+
+  public void testSingleCopyTasks(boolean runCopyTasksOnTarget)
```

Comment:
What happens if there are extra paths apart from table location in the parent path?